.github/workflows/release.yml

```
@@ -0,0 +1,97 @@
+      - "release/*"
+
+jobs:
+  release-java-modules:
+    name: Release Java Modules
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        module: [ 'sep490-common', 'sep490-idp' ]
+    steps:
+      - name: Extract version from branch name
+        run: echo "VERSION=${GITHUB_REF#refs/heads/release/}" >> $GITHUB_ENV
```

actionlint (1.7.3)

shellcheck reported issue in this script: SC2086:info:1:53: Double quote to prevent globbing and word splitting

(shellcheck)